### PR TITLE
Middle Finger of Vecna; Binder.json

### PR DESCRIPTION
**Pull request by tylermac**

### CODE REVIEW

File: class/Middle Finger of Vecna; Binder.json [hunk 1]
```diff
@@ -0,0 +1,5467 @@
...
+                                        ]
+                                    }
+                                ]
+                            }
+                        ],
+                        [
+                            {
+                                "name": "Anima Shield",
+                                "entries": [
+                                    "At 6th level, you force a bound vestige into being your personal watchdog. While you are bound to a vestige, you add your proficiency bonus to your initiative rolls."
```
[the remark this answers is not in the document]
Comment (TheGiddyLimit) [reply class/Middle Finger of Vecna; Binder.json]: It's not supported yet, is why. I'd honestly forgotten; it's never come up. I'll implement it tonight.

[the remark this answers is not in the document]
Comment (TheGiddyLimit) [reply class/Middle Finger of Vecna; Binder.json]: Changes should now be deployed, if you want take a crack at making the appropriate magic variants.

**Comment by tylermac** [comment]
Looking good.

The {@genericBonus} doesn't seem to show up on the Generic Variant description in the item list, though it does for the generated items. Interestingly it does show up for the giddy +4 sample file.

**Comment by TheGiddyLimit** [comment]
Any particular reason this was closed..? I was planning to investigate the `genericBonus` issue (although I suppose I'll do that anyway) before merging. It could've been merged regardless, honestly.

**Comment by tylermac** [comment]
I merged it into my fork, and out of habit deleted the branch. :|

**Comment by tylermac** [comment]
Oops again

**Comment by TheGiddyLimit** [comment]
Seems it's just an inconsistency with the way we render `variant` data; I'll clean that up.

**Comment by tylermac** [comment]
Ah, I completely missed that the entries was not in the same level as the example. :)

**Pull request merged**